discord/user.py

```diff
@@ -153,7 +153,7 @@ def public_flags(self) -> PublicUserFlags:
     def avatar(self) -> Optional[Asset]:
         """Optional[:class:`Asset`]: Returns an :class:`Asset` for the avatar the user has.
 
-        If the user does not have a traditional avatar, ``None`` is returned.
+        If the user has a non-traditional avatar or a global default discord avatar, ``None`` is returned.
```

Comment:
```suggestion
        If the user has not uploaded an avatar, ``None`` is returned.
```

Reply:
This is ambiguous since a user can upload a guild avatar but not his global one. I think you have to say 'global avatar'.

Reply:
```suggestion
        If the user has not uploaded a global avatar, ``None`` is returned.
```